README.md

```diff
@@ -113,6 +120,32 @@ To contribute:
 
 ### Development
 
+- Clone repository
+
+  - Generate your SSH keys as suggested [here](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+  - Clone the repository by typing (or copying) the following line in a terminal at your selected path in your machine:
+
+  ```
+  cd && mkdir -p $HOME/repositories/ && cd  $HOME/repositories/
+  git clone git@github.com:UCL-ARC/python-tooling.git
+  cd python-tooling
+  ```
+
+- To create and remove your virtual environment
+
+  ```
+  # conda create -n ptoolingVE python=3.11 pip -c conda-forge
```

Comment:
I think they will also need `tox` in their development environment to run the tests, or `pytest` if they want to run it manually